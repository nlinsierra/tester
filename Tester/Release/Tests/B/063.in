9C 6C 2D 6H QH 9D 3C
