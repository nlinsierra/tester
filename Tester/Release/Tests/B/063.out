45
2C 4C 5C 7C 8C TC JC QC KC AC 3D 4D 5D 6D 7D 8D TD JD QD KD AD 2H 3H 4H 5H 7H 8H 9H TH JH KH AH 2S 3S 4S 5S 6S 7S 8S 9S TS JS QS KS AS
